### PR TITLE
Styling the Politics Article

### LINKED ISSUE #41: Style Politics Article
Style politics article card layout.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Adjust the image, title and description sizes in the politics article to fit the card. 

## Purpose
Enables the user to see the introduction of  Politics article in the card in its proper size. 

Closes #41

### CODE REVIEW

File: the-toad-tribune/src/layout/Politics.tsx [hunk 1]
```diff
@@ -63,10 +77,43 @@ const PoliticsStyles = styled.div`
...
+  margin: .4rem;
+  
+`;
+
+const PoliticsPublishedDate = styled.h5`
+margin: 0 .4rem .2rem .4rem;
+`;
+
+const PoliticsDescription = styled.p`
+`;
```
Comment (Lap343) [inline the-toad-tribune/src/layout/Politics.tsx:117]: does this need to be here if there is no styling to it?

**Review by connielion — APPROVED** [review]
Nice job!

**Review by Lap343 — APPROVED** [review]
Looking good

File: the-toad-tribune/src/layout/Politics.tsx (hunk 1 above)
Comment (gonzalespaulb) [inline the-toad-tribune/src/layout/Politics.tsx:117]: You can also nest the H and P tags inside PoliticsContentContainer like how you would with SCSS. 

Example:
``` 
const PoliticsContentContainer = styled.div`
  min-height: 50%;
  min-width: 300px;
  display: flex;
  flex-direction: column;
  margin: .4rem .6rem .6rem .6rem;

         h3 {
                margin.3rem;
              }
`;

```

Comment (kmausolf) [reply the-toad-tribune/src/layout/Politics.tsx:117]: The team likely intends to add styling at a later point.  When we do this, it's a good idea to add a `// TODO: Add styling`

**Review by gonzalespaulb — APPROVED** [review]

File: the-toad-tribune/src/layout/Politics.tsx (hunk 1 above)
Comment (JayDampitan) [reply the-toad-tribune/src/layout/Politics.tsx:117]: Thank you for the tips everyone!

Comment (JayDampitan) [reply the-toad-tribune/src/layout/Politics.tsx:117]: Thanks tol!

Comment (krferrell) [reply the-toad-tribune/src/layout/Politics.tsx:117]: Nice. Good to know. That will tidy things up quite a bit.

**Pull request merged**